sky/packages/sky/lib/src/widgets/ink_well.dart

```diff
@@ -33,11 +35,12 @@ class InkSplash {
     _performance = new ValueAnimation<double>(
       variable: _radius,
       duration: new Duration(milliseconds: (_targetRadius / _kSplashUnconfirmedVelocity).floor())
-    )..addListener(_handleRadiusChange)
-     ..play();
+    )..addListener(_handleRadiusChange);
+
+    // Wait kTapTimeout to avoid creating tiny splashes during scrolls.
+    _startTimer = new Timer(kTapTimeout, _play);
```

Comment:
Yeah, I started down that path, but it required even more changes because ink splash wants the "down" location in local coordinates.  Let's work up to that in another iteration.